api/v1alpha2/types.go

```diff
@@ -169,6 +169,10 @@ type NetworkSpec struct {
 	// Subnets configuration.
 	// +optional
 	Subnets Subnets `json:"subnets,omitempty"`
+
+	// ControlPlaneLoadBalancerScheme (defaults to Internet-facing)
+	// +optional
+	ControlPlaneLoadBalancerScheme ClassicELBScheme `json:"controlPlaneLoadBalancerScheme,omitempty""`
```

Comment:
Let’s make this a pointer given that’s optional

Reply:
I've made this a pointer in my fork, but I'm having trouble figuring out how to make the change reflect here